libs/watt/src/lib/components/table/watt-table.component.html

```diff
@@ -75,7 +77,9 @@
     [ngClass]="{
       'watt-table-highlight-row': !disabled && !suppressRowHoverHighlight,
       'watt-table-clickable-row': !disabled && rowClick.observed,
-      'watt-table-active-row': _isActiveRow(row)
+      'watt-table-active-row': _isActiveRow(row),
+      'watt-table-hide': true,
+      'watt-table-no-end-divider': hideColumnHeaders
```

Comment:
suggestion: doesn't look like `watt-table-hide` is used 🤔

```suggestion
      'watt-table-active-row': _isActiveRow(row),
      'watt-table-no-end-divider': hideColumnHeaders
```